roles/openshift_node/tasks/main.yml

```diff
@@ -77,7 +77,7 @@
 - name: Check scheduleable state
   delegate_to: "{{ openshift_first_master }}"
   command: >
-    {{ openshift.common.client_binary }} get node {{ inventory_hostname }}
+    {{ openshift.common.client_binary }} get node {{ openshift_hostname }}
```

Comment:
I definitely prefer this.  I'm curious why we don't already have it though.

Would it be better to make a change likt this here? https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/tasks/main.yml#L25